src/pages/Helper.js

```diff
@@ -0,0 +1,5 @@
+export const COLORS = {
```

Comment:
I think the helper file could be moved inside the components directory since it is related to a component and not a page.
+1 Could you rename the constant to a more specific name? like DRAGGABLE_ITEM_COLORS